### PR TITLE
Use new validation for creating/updating triggers

### PR DESCRIPTION
**Pull request by Dimedrolity**
After https://github.com/moira-alert/moira/pull/817

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, 3.10, 3.11]
```
Comment (Dimedrolity) [inline .github/workflows/test.yml]: Do we not need support 3.6 and 3.7?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: [3.8, 3.9, 3.10, 3.11]
+        python-version: [3.8, 3.9, 3.10.10, 3.11]
```
Comment (Dimedrolity) [inline .github/workflows/test.yml]: Seems strange but it works

File: .github/workflows/test.yml (hunk 1 above)
Comment (kotbauk) [reply .github/workflows/test.yml]: Github doesn't support 3.6 and 3.7 in ubuntu-22.04 vm

File: .github/workflows/test.yml (hunk 2 above)
Comment (kotbauk) [reply .github/workflows/test.yml]: Github convert 3.10 to 3.1, so 3.10 doesn't work (probably 3.10.10 converted to 3.10.1 bl bl)

File: moira_client/models/trigger.py [hunk 3]
```diff
@@ -149,13 +150,13 @@ def _send_request(self, trigger_id=None):
             api_response = TriggerManager(self._client).fetch_by_id(trigger_id)
 
         if trigger_id and api_response:
-            res = self._client.put('trigger/{id}'.format(id=trigger_id), json=data)
+            res = self._client.put('trigger/{id}?{validate}'.format(id=trigger_id, validate=self.QUERY_PARAM_VALIDATE_FLAG), json=data)
```
Comment (Tetrergeru) [inline moira_client/models/trigger.py]:
```suggestion
            res = self._client.put(f'trigger/{trigger_id}?{self.QUERY_PARAM_VALIDATE_FLAG}', json=data)
```

**Review by Tetrergeru — DISMISSED** [review]

File: moira_client/models/trigger.py (hunk 3 above)
Comment (Dimedrolity) [reply moira_client/models/trigger.py]: Thx, done https://github.com/moira-alert/python-moira-client/pull/26/commits/f5b2e119e3bf344fdcef777da764954b2117385f

**Review by Tetrergeru — DISMISSED** [review]

**Review by Tetrergeru — APPROVED** [review]

**Pull request merged**